service/internal/configunmarshaler/common.go

```diff
@@ -0,0 +1,30 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I'd expect these in a file named `error.go` instead of common

Reply:
Done.